web/yo/app/scripts/services/mainutils.js

```diff
@@ -528,6 +530,31 @@ angular.module('oncokbApp')
             var date = new Date();
             return date.getTime();
         }
+        function getAllUUIDs() {
+            return allUUIDs;
+        }
+        function setAllUUIDs(userName, uuids) {
```

Comment:
This function is serving different purposes. I suggest separating the functionalities.